docs/data/sql_functions.yml

```diff
@@ -350,6 +350,17 @@ string:
   - sql: LOCATE(string1, string2[, integer])
     table: STRING1.locate(STRING2[, INTEGER])
     description: Returns the position of the first occurrence of string1 in string2 after position integer. Returns 0 if not found. Returns NULL if any of arguments is NULL.
+  - sql: URL_DECODE(string)
+    table: STRING.urlDecode()
+    description:
+      Decodes a given string in 'application/x-www-form-urlencoded' format using the UTF-8 encoding scheme, will be null if input is null.
+      If there is an issue with the decoding process, such as encountering an illegal escape pattern, the function returns the original input value.
+      If the encoding scheme is not supported, an error will be reported.
```

Comment:
Can we have a test for this?

I guess we need to pass a string like `_UTF16’Hello` or `_UTF32’Hello` and may be with usage of some non UTF8 symbols

Reply:
So should I construct a string encoded with UTF_16 to test?